engines/unknown-game-dialog.cpp

```diff
@@ -0,0 +1,144 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "common/translation.h"
+#include "common/str-array.h"
+#include "gui/gui-manager.h"
+#include "gui/message.h"
+#include "gui/ThemeEval.h"
+#include "gui/widgets/popup.h"
+#include "engines/unknown-game-dialog.h"
+#include "backends/platform/sdl/sdl.h"
+
+namespace GUI {

```

Comment:
I know you initially added the file to `gui/`, and it made sense to use the GUI namespace then, but since now the file is in `engines/` I don't think you should use the GUI namespace. Looking at the other dialogs in `engines/dialogs.h` they are in the global namespace.

So I would suggest to do the same and remove this line, then remove the closing bracket at the end of this file, and do the same in unknown-game-dialog.h.